src/components/Modals/Settings/ClearCache.tsx

```diff
@@ -30,13 +33,21 @@ const ClearCacheButton = ({
   label,
   tooltipText,
   onClick,
+  isDrawer = false,
```

Comment:
This is my only concern of this PR, I hate those props, it doesn't make sense from a UI point of view to differenciate using a prop

But it will be removed when the wallet drawer will be enabled and assume we are in a drawer 100% of the time

Reply:
As discussed in DMs,  keeping as-is as a hacky simple version for now - once we tackle mobile drawer too (as you've noted) then this can go away